dnf/cli/cli.py

```diff
@@ -100,6 +100,8 @@ def _list_cmd_calc_columns(output, ypl):
 
 
 def cachedir_fit(conf):
+    if conf.releasever is None:
+        conf.releasever = dnf.rpm.detect_releasever('/')
```

Comment:
I'm affraid that it might be confusing that cachedir_fit modifies the config. Is it really the best place?